src/css/typography.css

```diff
@@ -1,25 +1,38 @@
 @font-face {
-  font-family: "gilroy";
+  font-family: "gilroy-normal";
```

Comment:
Shouldn't all there have the same `font-familiy`?

Reply:
Done

Reply:
Let's talk about this next week.
I mean all gilroy `font-face`s should have the same font-family name (`gilroy`)